commands/host.vegman

```diff
@@ -205,7 +205,7 @@ function cmd_virtualmedia_mount {
 # @restrict cmd_virtualmedia_umount admin
 # @doc cmd_virtualmedia_umount
 # Unmount the virtual media
-#  -y, --yes  - does not ask for interactive confirmation
+#  -y, --yes  - do not ask for interactive confirmation
```

Comment:
While you're at it, please change it to capitalized 'Do'. Also, please make this refactoring a separate commit (and please check in other files where `--yes` is specified. Let's make the messge identical everywhere).